acceptance/tests/pxp-module-puppet/run_puppet_result_changed.rb

```diff
@@ -0,0 +1,60 @@
+require 'pxp-agent/test_helper.rb'
+
+ENVIRONMENT_NAME = 'changed_result'
+MODULE_NAME = 'changed'
```

Comment:
I think MODULE_NAME is leftover from a previous swing?   Nothing is using it now.